core/ledger/kvledger/txmgmt/rwsetutil/rwsetV1ProtoBytes.txt

```diff
@@ -0,0 +1,21 @@
+K
```

Comment:
Can you declare this file as a binary file. So that git does not interfere with the contents of this file while transferring across platforms